### ISSUE TITLE
Migrate to Pantheon: WordPress Doc Update 

### ISSUE THREAD

**Issue by tinefin** [pantheon-systems/documentation#1862]
Re: [Migrate to Pantheon: WordPress](https://pantheon.io/docs/migrate-wordpress)
## Issue Description:
There are now only 2 options for import: Url for existing site or manual migration. Sites that are not yet LIVE therefore only have the option of manual migration (no archive upload anymore).

## Suggested Resolution 
In this section:
"The following scenarios are exceptions to the recommended process and require manually migrating the site:"

We should add a bullet point with something like: 
- Site is not yet live on a url / only exists on local machine.

**Comment by rachelwhitton** [comment]
WordPress users can import archives via URL (within file size limits) using Terminus:

```
##NAME
    terminus sites import


##DESCRIPTION
    Import a new site

##SYNOPSIS
    [--url=<url>] [--name=<name>] [--site=<site>] [--label=<label>] [--org=<id>]

##OPTIONS
    --url=<url>
        URL of archive to import
    --name=<name>
        (deprecated) use --site instead
    --site=<site>
        Name of the site to create (machine-readable)
    --label=<label>
        Label for the site
    --org=<id>
        UUID of organization into which to add this site
```

Tests are successful. @tinefin what do you think about recommending Terminus first, then directing to the manual migration process for archives exceeding file size limits?

**Comment by rachelwhitton** [comment]
Closed by #1872
